app/models/group.rb

```diff
@@ -414,6 +414,16 @@ def bulk_add(user_ids)
       if self.title.present?
         User.where(id: user_ids).update_all(title: self.title)
       end
+
+      if self.grant_trust_level.present?
+        User.where(id: user_ids).find_each do |user|
+          if user.trust_level < self.grant_trust_level
+            user.change_trust_level!(self.grant_trust_level)
+            user.trust_level_locked = true
```

Comment:
We're already [doing this](https://github.com/discourse/discourse/blob/5dbd6a304bed5400be481d71061d3e3ebb4d6785/app/models/group_user.rb#L60) when adding individual user to a group.